Sitka-Sound-Science-Center/sitka-landslide
# Add resources list on each resources page

## Overview

Make it easier to navigate between resources by adding a list of all resources to the bottom of each resources page. This was requested by the client.

### Demo

<img width="542" alt="Screen Shot 2022-05-24 at 9 27 01 PM" src="https://user-images.githubusercontent.com/1809908/170158953-eb8698bd-8eb9-450c-b3ce-e2bed55fae11.png">


### Notes

I went with the "list all of them" over the dropdown option because it seemed easier. I think it will work nicely on mobile to have them expanded by default.

## Testing Instructions

 * Run the application
 * Open each resource
 * Confirm that each resource page contains a list of all resources at the bottom, like in the screenshot

## Checklist
